inferentia/README.md

```diff
@@ -47,33 +47,44 @@ After logging into the inf1* instance, you will need to clone
 or simply clone with https.
 Clone this repo with Github to home repo `/home/ubuntu`.
 
+Ensure that the neuron runtime 1.0 demon (neuron-rtd) is not running and set up
+and install neuron 2.X runtime builds with
+```
+ sudo ./python_backend/setup-pre-container.sh
+```
+
 Then, start the Triton instance with:
 ``` 
-docker run -v /home/ubuntu/python_backend:/home/ubuntu/python_backend -v /lib/udev:/mylib/udev -v /run:/myrun --shm-size=1g --ulimit memlock=-1 -p 8000:8000 -p 8001:8001 -p 8002:8002 --ulimit stack=67108864 -ti nvcr.io/nvidia/tritonserver:<xx.yy>-py3
+docker run --device /dev/neuron0 <more neuron devices> -v /home/ubuntu/python_backend:/home/ubuntu/python_backend -v /lib/udev:/mylib/udev --shm-size=1g -e "AWS_NEURON_VISIBLE_DEVICES=ALL" --ulimit memlock=-1 -p 8000:8000 -p 8001:8001 -p 8002:8002 --ulimit stack=67108864 -ti nvcr.io/nvidia/tritonserver:<xx.yy>-py3
+```
+Note 1: The user would need to list any new neuron device to run during container initialization.
+For example, to use 4 neuron devices on an instance, the user would need to run with:

```

Comment:
Isn't there a way to launch container with an access to all Neuron cores?

Reply:
No not right now.